removed normalized_avg_processors and created documentation on how to add it back (#1981)
ubccr/xdmod: docs/upgrade.md

Make sure to update `/etc/httpd/conf.d/xdmod.conf` to change
`/opt/xdmod-{{ page.prev_sw_version }}` to `/opt/xdmod-{{ page.sw_version }}`.

> 
> ### Configuration File Changes
> 
> The "Job Size: Normalized" statistic in the Jobs realm is removed from the
> default list of statistics. See [this
> page](howto-normalized-avg-processors.html) for more information.
> 
> 
> Additional 11.5.0 Upgrade Notes
> -------------------

Reviewer: ```suggestion
Additional 11.5.0 Upgrade Notes
-------------------

### Configuration File Changes

The "Job Size: Normalized" statistic in the Jobs realm is removed from the
default list of statistics. See [this
page](howto-normalized-avg-processors.html) for more information.
```
The changes will apply when they upgrade to 11.5.0 so this should go in the 11.5.0 section.

[github-release]: https://github.com/ubccr/xdmod/releases/tag/v{{ page.rpm_version }}

> [mysql-config]: configuration.html#mariadb-configuration

Reviewer: These two lines should be the last in the file.

Author: changed it

> Configuration File Changes
> ----------------------------

Reviewer: ```suggestion
### Configuration File Changes
```

> - Normalized Avg Processors Statistics was removed and in order to re add the statistics
>   please see the [Restore the "Job Size: Normalized" Statistic](howto-normalized-avg-processors.html)

Reviewer: ```suggestion
The "Job Size: Normalized" statistic in the Jobs realm is removed from the
default list of statistics. See [this
page](howto-normalized-avg-processors.html) for more information.
```
